task/mgmt-gateway/src/update_buffer.rs

```diff
@@ -0,0 +1,148 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+use crate::{Log, __RINGBUF};
+use core::marker::PhantomData;
+use gateway_messages::ResponseError;
+use ringbuf::ringbuf_entry;
```

Comment:
Nit: instead of importing `__RINGBUF`, you could
```
use ringbuf::ringbuf_entry_root as ringbuf_entry
```
(which logs to a ringbuf in the root of the crate)

Reply:
TIL! Thanks